app/models/meeting.rb

```diff
@@ -7,7 +7,11 @@ def partner_id
     user_meetings.find_by(is_requestor: false).user_id
   end
 
-  def user_id
+  def host_id
```

Comment:
Changing it to host_id so that it's easier to see who the actual creator of the meeting is. Nice!